server/handler.go

```diff
@@ -601,3 +602,22 @@ func (h *Handler) GetIncorrectNamespaceRegions() ([]*core.RegionInfo, error) {
 	}
 	return c.cachedCluster.GetRegionStatsByType(incorrectNamespace), nil
 }
+
+// FieldError connects an error to a particular field
+type FieldError struct {
+	error
+	field string
+}
+
+// ParseVarUint wraps strconv.ParseUint with detailed error reporting
+func ParseVarUint(vars map[string]string, varName string, base int, bitsize int) (uint64, error) {
```

Comment:
Why put it in package `server`? Seems `server/api` or `pkg/apiutil` could be a better place.

Reply:
Generally its better to place things lower down in the dependency graph if it doesn't take in additional dependencies itself. So it seems like I should put things into apiutil that don't require ErrCode, assuming that the vars convention is not particular to our handlers.